Doc/library/itertools.rst

```diff
@@ -48,6 +48,7 @@ Iterator            Arguments               Results
 Iterator                        Arguments                       Results                                             Example
 ============================    ============================    =================================================   =============================================================
 :func:`accumulate`              p [,func]                       p0, p0+p1, p0+p1+p2, ...                            ``accumulate([1,2,3,4,5]) --> 1 3 6 10 15``
+:func:`batched`                 p, n                            [p0, p1, ..., p_n-1], ...                           ``batched('ABCDEFG') --> ABC DEF G``
```

Comment:
Looks like there's a missing `n=3` argument in the example.

Reply:
Thanks for noticing this.